spring-cloud-dataflow-server-local/src/main/resources/application.yml

```diff
@@ -10,7 +10,7 @@ spring:
       registry:
         populator:
           locations: classpath:META-INF/${binder}-apps.properties
-
+serverStore: redis
```

Comment:
Yeah, I agree. I thought of having this as temporary until we move away from `redis`. But, that is not necessary as we can rely on H2 and use the default. I will update the PR to move the `serverStore` property completely. Also, we need to decide whether we really need the in-memory implementation of these repositories as they are never used.